felix/bpf/conntrack/bpf_scanner.go

```diff
@@ -71,9 +74,14 @@ func registerConntrackMetrics() {
 // Note: the tests for this object are largely in the bpf/ut package, since
 // we require a privileged environment to test the BPF program.
 type BPFProgLivenessScanner struct {
-	ipVersion int
-	timeouts  Timeouts
-	logLevel  BPFLogLevel
+	ipVersion                    int
+	timeouts                     Timeouts
+	logLevel                     BPFLogLevel
+	scaling                      bool
```

Comment:
```suggestion
	autoScale                      bool
```
think this would be more guessable